packages/module/patternfly-docs/content/extensions/chatbot/examples/Messages/MessageWithFeedback.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import Message from '@patternfly/chatbot/dist/dynamic/Message';
+import patternflyAvatar from './patternfly_avatar.jpg';
+import { Checkbox, FormGroup, Stack } from '@patternfly/react-core';
+
+export const MessageWithFeedbackExample: React.FunctionComponent = () => {
+  const [hasCloseButton, setHasCloseButton] = React.useState(true);
+  const [hasTextArea, setHasTextArea] = React.useState(false);
+
+  return (
+    <>
+      <Stack hasGutter>
+        <FormGroup role="radiogroup" isInline fieldId="feedback-card" label="Variant">
+          <Checkbox
+            isChecked={hasTextArea}
+            onChange={() => {
+              setHasTextArea(!hasTextArea);
+            }}
+            name="basic-inline-radio"
+            label="Has text area"
+            id="has-text-area"
+          />
+        </FormGroup>
+        <Message
+          name="Bot"
+          role="bot"
+          avatar={patternflyAvatar}
+          content="Bot message with feedback form only"
+          userFeedbackForm={{
+            quickResponses: [
+              { id: '1', content: 'Correct' },
+              { id: '2', content: 'Easy to understand' },
+              { id: '3', content: 'Complete' }
```

Comment:
To integrate conversation design into the microcopy from the get-go, wondering if we can change these to align with the phrasing I'd like consumers to default to? Would these button labels be too long? Idk if I love these suggestions either..I'm trying to envision how the buttons are used in practice since I haven't seen this pattern before 🤔

```suggestion
              { id: '1', content: 'Helpful information' },
              { id: '2', content: 'Easy to understand' },
              { id: '3', content: 'Resolved my issue' }
```